lib/tasks/update_banned_url_shorteners.rake

```diff
@@ -0,0 +1,35 @@
+# typed: false
+
+require "open-uri"
+
+desc "Update banned list of URL shorteners"
+
+task update_banned_url_shorteners: :environment do
+  url = "https://raw.githubusercontent.com/mayakyler/link-shorteners/refs/heads/main/js-link-shorteners/src/link-shorteners.txt"
+
+  begin
+    response = URI.parse(url).open
+    content = response.read
+
+    domains = content.split("\n").reject { |line| line.empty? || line.starts_with?("#") }
+
+    banned_by_user = User.find_by(username: "pushcx")
```

Comment:
Hmm. This isn’t going to work for sister sites using our codebase and I don’t have a better idea for it. Maybe a new setting in the site-wide settings at the end of `config/application.rb`?